### ISSUE TITLE
Camera app : "remove EXIF data after capture" defaults to "on" ?

### ISSUE THREAD

**Issue by Saroumane** [GrapheneOS/os-issue-tracker#1036]
**System info:**

Google Pixel 3a
Build number SQ1A.220205.005.2022030501

I just discovered that I have roughly 1 month of pictures taken with GrapheneOS Camera app without EXIF data (most annoyingly : without Exposure Date). The last "good" picture I have is from the 8th of February.
I guess it's because of the new setting "remove EXIF data after capture" that defaults to "on" in the Camera app. (Or maybe I put it "on" inadvertently ?)
As my sharing app (Signal) already takes care of EXIF data, I put this setting back to "off" : I do need "Exposure date" of pictures for my PC storage to stay consistent.
Now I guess the only workaround I can do is retag all pictures manually, using the filename as a hint for Exposure Date.

In my humble opinion, such a big change could have been listed here : https://grapheneos.org/releases#changelog
(Of course if the setting defaults to "off", that's entirely up to me to be more careful :)

**Comment by Zanthed** [comment]
This is a privacy and security focused app created by a privacy and security focused operating system.

It has been heavily requested to remove EXIF by default and there are no plans to revert that.

> In my humble opinion, such a big change could have been listed here : https://grapheneos.org/releases#changelog

That is for the operating system release notes, not the app release notes.

It was announced in version 11 that EXIF removal is now default. https://github.com/GrapheneOS/Camera/releases/tag/11

**Comment by Saroumane** [comment]
> This is a privacy and security focused app created by a privacy and security focused operating system.
> 
> It has been heavily requested to remove EXIF by default and there are no plans to revert that.
> 
> > In my humble opinion, such a big change could have been listed here : https://grapheneos.org/releases#changelog
> 
> That is for the operating system release notes, not the app release notes.
> 
> It was announced in version 11 that EXIF removal is now default. https://github.com/GrapheneOS/Camera/releases/tag/11

That's not fair, there are "app level" changes listed in the system release notes.

One random pick : "stop enforcing camera sound when using a carrier based in India/Japan/Korea since we aren't required to enforce it and is trivially bypassed in multiple ways so it's nothing more than an annoyance"
(which is arguably less important that the change regarding EXIF data)

Regarding my use case not fitting the most requested, fine by me.

**Comment by flawedworld** [comment]
>One random pick : "stop enforcing camera sound when using a carrier based in India/Japan/Korea since we aren't required to enforce it and is trivially bypassed in multiple ways so it's nothing more than an annoyance"

This is not an app level change and this is referring to the infrastructure within the OS to enforce this.

**Comment by Zanthed** [comment]
> One random pick : "stop enforcing camera sound when using a carrier based in India/Japan/Korea since we aren't required to enforce it and is trivially bypassed in multiple ways so it's nothing more than an annoyance"

Not an app level change. https://github.com/GrapheneOS/platform_frameworks_base/commit/de567a637a2ccf0cd25de4d473605f0735436cd9

**Comment by thestinger** [comment]
We link the release notes for our standalone app releases in the OS release notes. If you want to read all those changes you need to read each of the linked release notes pages too.

**Comment by Saroumane** [comment]
> > One random pick : "stop enforcing camera sound when using a carrier based in India/Japan/Korea since we aren't required to enforce it and is trivially bypassed in multiple ways so it's nothing more than an annoyance"
> 
> Not an app level change. [GrapheneOS/platform_frameworks_base@de567a6](https://github.com/GrapheneOS/platform_frameworks_base/commit/de567a637a2ccf0cd25de4d473605f0735436cd9)

Hello, I won't debate about user's point of view (which is "function oriented") versus dev's point of view (which is code oriented). Let's just say that in a perfect world such a novelty should be introduced by a "one-time screen". But we are not in a perfect world and ressources are scarce, aren't they ?

More concerning : this setting is not always honored.
I deactivated "Remove EXIF data after capture" on the 6th of March.
Since that moment I took 362 pictures. (I always invoke Camera the same way, by double pressing the top right physical button)
Among them 11 pictures (taken on 4 different days) have had their EXIF data completely removed.
Can I open a bug report about this ?
